README.md

```diff
@@ -169,9 +155,9 @@ Distributed under the Apache-2.0 License. See `LICENSE` for more information.
 
 ## Contact
 
-Your Name - [@twitter_handle](https://twitter.com/twitter_handle) - email
+Tiago Mendonça Fernandes - tiago.fernandes@elecnor.es
```

Comment:
Maybe we should replace the email with the team-generic one: [EOEPCA mailbox](eoepca.systemteam@telespazio.com)

Reply:
Changed.